pkg/eval/eval.go

```diff
@@ -0,0 +1,28 @@
+package eval
+
+import (
+	"github.com/k14s/ytt/pkg/files"
+	"github.com/k14s/ytt/pkg/yamlmeta"
+	"go.starlark.net/starlark"
+)
+
+type Loader interface {
```

Comment:
since TemplateLoader sits within workspace for now, are we able to combine pkg/eval with pkg/workspace? i might separate them eventually but it seems to be a scope creep for this PR.